bin/benchpark

```diff
@@ -42,20 +42,50 @@ def benchpark_list(subparsers, actions_dict):
 
 def benchpark_list_handler(args):
     source_dir = source_location()
+
+    print("Benchmarks:")
+    experiments_dir = source_dir / "experiments"
+    for x in os.listdir(experiments_dir):
+        for y in os.listdir(experiments_dir / x):
+            print(f"\t{x}/{y}")
```

Comment:
I recommend abstracting this into a function that maps benchmark identifiers to paths (since this is now copied in two places, consolidating into one function would be helpful).

(same for system)

Reply:
Separated out into a function, but not sure what mapping you were interested in, I just have a list.